app/controllers/admin/divisions_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Admin::DivisionsController < Admin::AdminController
+  before_action :find_division, only: [:show, :update, :destroy]
```

Comment:
My expectation was that we would only use these for the public site for now -- don't wanna muck too much with the behavior of the admin panel right now. I thought we may need to use this so that we could find divisions by the kinds of symbols that we get back from the public frontend (`:us`, `:argentina`, `:nicaragua`), especially if `friendly_id` supported i18n, but its i18n support seems pretty minimal, so it might not be worth using for this just now.